restapi/src/main/java/io/stargate/web/resources/v2/RowsResource.java

```diff
@@ -693,14 +706,19 @@ private Object getRows(
     return raw ? rows : new GetResponseWrapper(rows.size(), newPagingState, rows);
   }
 
-  private List<ColumnOrder> buildSortOrder(String sort)
-      throws com.fasterxml.jackson.core.JsonProcessingException {
+  private List<ColumnOrder> buildSortOrder(String sort) {
     if (Strings.isNullOrEmpty(sort)) {
       return new ArrayList<>();
     }
 
     List<ColumnOrder> order = new ArrayList<>();
-    Map<String, String> sortOrder = mapper.readValue(sort, Map.class);
+    Map<String, String> sortOrder;
+    try {
+      sortOrder = mapper.readValue(sort, new TypeReference<HashMap<String, String>>() {});
```

Comment:
Is it important to bind into `Map<String, String>` instead of `Map<String, Object>`? Not a big deal but does coerce numbers into Strings and would fail if values JSON Arrays, Objects.

Reply:
These will be purely strings for example `firstName ASC` or `username DESC`. If there is a number provided then it's not a big deal to handle as a string.

Reply:
I guess I was more thinking of whether it could just use that same `Map<String, Object>` mapping as other methods, if a helper method was extracted. Sounds like it should only ever get String values, so in happy path no difference.

Reply:
I ended up switching to `Map<String, Object>` so I could reuse the catch block.